app/views.py

```diff
@@ -42,6 +42,27 @@ def getUser():
     return user
 
 
+def getProjects(username):
+    if not isLoggedIn():
+        return [{'error': "Not logged in."}]
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+    projects = ts.get_projects()
+
+    userProjects = []
+
+    # Get each project the user has access to
+    for project in projects:
+        if 'users' in project and username in project['users']:
+            userProjects.append(project)
+        elif 'owner' in project and username == project['owner']:
```

Comment:
I thought we deprecated the owner field on projects in favor of project level permissions (member, spectator, manager). I think we can drop this check.